docs/docs/configuration/cameras.md

```diff
@@ -0,0 +1,407 @@
+
+Each role can only be assigned to one input per camera. The options for roles are as follows:
+
+| Role     | Description                                                                          |
+| -------- | ------------------------------------------------------------------------------------ |
+| `detect` | Main feed for object detection                                                       |
+| `clips`  | Clips of events from objects detected in the `detect` feed. [docs](#recording-clips) |
+| `record` | Saves 60 second segments of the video feed. [docs](#247-recordings)                  |
+| `rtmp`   | Broadcast as an RTMP feed for other services to consume. [docs](#rtmp-streams)       |
+
+### Example
+
+```yaml
+mqtt:
+  host: mqtt.server.com
+cameras:
+  back:
+    ffmpeg:
+      inputs:
+        - path: rtsp://viewer:{FRIGATE_RTSP_PASSWORD}@10.0.10.10:554/cam/realmonitor?channel=1&subtype=2
+          roles:
+            - detect
+            - rtmp
+        - path: rtsp://viewer:{FRIGATE_RTSP_PASSWORD}@10.0.10.10:554/live
+          roles:
+            - clips
+            - record
+    width: 1280
+    height: 720
+    fps: 5
+```
+
+## Masks & Zones
+
+### Motion masks
+
+Masks are used to ignore initial detection in areas of your camera's field of view.
+
+To create a poly mask:
+
+1. Visit the [web UI](/usage/web)
+1. Click the camera you wish to create a mask for
+1. Click "Mask & Zone creator"
+1. Click "Add" on the type of mask or zone you would like to create
+1. Click on the camera's latest image to create a masked area. The yaml representation will be updated in real-time
+1. When you've finished creating your mask, click "Copy" and paste the contents into your `config.yaml` file and restart Frigate
+
+Example of a finished row corresponding to the below example image:
+
+```yaml
+motion:
+  mask: '0,461,3,0,1919,0,1919,843,1699,492,1344,458,1346,336,973,317,869,375,866,432'
+```
+
+![poly](/img/example-mask-poly.png)
+
+```yaml
+# Optional: camera level motion config
+motion:
+  # Optional: motion mask
+  # NOTE: see docs for more detailed info on creating masks
+  mask: 0,900,1080,900,1080,1920,0,1920
+```
+
+### Zones
+
+Zones allow you to define a specific area of the frame and apply additional filters for object types so you can determine whether or not an object is within a particular area. Zones cannot have the same name as a camera. If desired, a single zone can include multiple cameras if you have multiple cameras covering the same area by configuring zones with the same name for each camera.
+
+During testing, `draw_zones` should be set in the config to draw the zone on the frames so you can adjust as needed. The zone line will increase in thickness when any object enters the zone.
+
+To create a zone, follow the same steps above for a "Motion mask", but use the section of the web UI for creating a zone instead.
+
```

Comment:
I took some liberties to update these two sections a little to point to the web UI